### DISCUSSION TITLE
Using go2rtc to create a 'lite' stream

### DISCUSSION THREAD

**Discussion by blacknell** [blakeblackshear/frigate#5409] [General]
My tablet in Home Assistant has very modest processing power (that's putting it kindly). For almost everything it does it'd fine and responsive.

However, when I use a front door camera detection automation to ask it to switch views to one with an embedded video stream of the camera in question it takes forever (say 20s). This makes it rather useless since I want to glance at it when I heard the audio alert on my way to the door. The video it eventually streams is glitchy too.

Is it possible use go2rtc to create a new lightweight stream perhaps with a very low frame rate/cheap codec to overcome this?

**Comment by blakeblackshear** [comment]
Yes. You can add whatever streams you want to go2rtc. There is no requirement to use them in frigate at all. Take a look at the ffmpeg module section of the go2rtc docs.

**Comment by blacknell** [comment]
I guess I’m wondering more about the stream from go2rtc.If I can define an output stream that has a low decoding overhead for my tablet then I’m hoping it will be more responsive. I appreciate that my pi will be doing work to transcode just for this but that’s fine. I’m in the dark as what output stream settings I should define. Any suggestions?RegardsPaulOn 7 Feb 2023, at 12:38, Blake Blackshear ***@***.***> wrote:﻿
Yes. You can add whatever streams you want to go2rtc. There is no requirement to use them in frigate at all. Take a look at the ffmpeg module section of the go2rtc docs.

**Reply by NickM-27** [reply]
It's all in the go2rtc docs https://github.com/AlexxIT/go2rtc#source-ffmpeg

You can lower the resolution, bitrate, encoding profile, etc
